docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 services:
   balatro:
-    image: devkitpro/devkitarm:latest
-    working_dir: /balatro
+    build:
+      dockerfile: ./Dockerfile
```

Comment:
I don't know much about apple containers. So correct me if I'm wrong here, but I don't think there should be separate Dockerfile outside of this compose file.

Mainly I say that because the new Dockerfile doesn't add any functionality to the compose file and just moves the declaration of the `image`/`workdir`/`cmd` variables to itself. It seems like a messy solution to target a niche audience of Apple Container users. 

I suggest just sticking to regular Docker compose for now, unless you can avoid modifying the compose file and adding a new Dockerfile. 

I'm all ears though, I have been and will continually be wrong in many things, haha.

Reply:
I'm sorry for looking at this so late. 

I mainly added that Dockerfile to use it with `container` since it doesn't support docker-compose.yml files. And I modified the docker-compose.yml file to use the Dockerfile because as you mentioned it only moved the declaration of the `image`, `workdir` and `cmd`, but it can also be left as-is.

Reply:
It's good work! I'm not a maintainer, so I don't get the final say. If it works and they like it, sweet!

My piece is, apple containers are suited more for capturing apple specific environments (containers with macos dev environments natively running on their Arm silicon, for example), so maybe we shouldn't introduce it as it has to add that dockerfile and it's use case doesn't fit here. 

However, nitpicking has it's limits! I'm not a mac dev, so maybe apple containers are badass and I'm just a grump.